src/main/java/cc/polyfrost/oneconfig/utils/commands/annotations/SubCommandGroup.java

```diff
@@ -31,19 +31,21 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-/**
- * Marks the name of a parameter.
- *
- * @see Main
- * @see Command
- */
+/** Use this annotation on subclasses to mark them as subcommands. This uses the value field and the aliases for the syntax. */
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ElementType.PARAMETER})
-public @interface Name {
+@Target(ElementType.TYPE)
+public @interface SubCommandGroup {
```

Comment:
can we name this something else

Reply:
sure this was diamonds idea

Reply:
how about `SubGroup`, `Subcommand`, or simply `Sub`? @Wyvest

Reply:
> how about `SubGroup`, `Subcommand`, or simply `Sub`? @Wyvest

actually this is fine